### PR TITLE
OCPQE-22232-add-prow-debug-vsphere5

### PR DESCRIPTION
**Pull request by weinliu**
/hold

### CODE REVIEW

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.16-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.16-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.16-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.16-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort


/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
/pj-rehearse abort

/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi

**Comment by weinliu** [comment]
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58307/rehearse-58307-pull-ci-openshift-openshift-tests-private-release-4.17-debug-winc-vsphere-ipi/1853766334370811904

18 case passed from :` ~ChkUpgrade&;~ConnectedOnly&;Smokerun&`

: OCP-73595:weinliu:Windows_Containers:[sig-windows] Windows_Containers Verify Display of Filesystem Graphs (metrics) for Windows Nodes [Serial] | 4s
-- | --
: OCP-37362:rrasouli:Windows_Containers:[sig-windows] Windows_Containers [wmco] wmco using correct golang version | 4s
: OCP-33779:sgao:Windows_Containers:[sig-windows] Windows_Containers Retrieving Windows node logs | 7s
: OCP-39858:rrasouli:Windows_Containers:[sig-windows] Windows_Containers Windows servicemonitor and endpoints check [Slow][Serial][Disruptive] [Serial] | 16m28s
: OCP-73752:weinliu:Windows_Containers:[sig-windows] Windows_Containers Monitor Network In, and Network Out graphs for Windows Pods managed by wmco | 4s
: OCP-33768:sgao:Windows_Containers:[sig-windows] Windows_Containers NodeWithoutOVNKubeNodePodRunning alert ignore Windows nodes | 4s
: OCP-39030:sgao:Windows_Containers:[sig-windows] Windows_Containers Re queue on Windows machines edge cases [Slow][Disruptive] [Serial] | 10m56s
: OCP-38188:jfrancoa:Windows_Containers:[sig-windows] Windows_Containers Get Windows instance/core number and CPU arch | 6s
: OCP-50403:jfrancoa:Windows_Containers:[sig-windows] Windows_Containers wmco creates and maintains Windows services ConfigMap [Disruptive] [Serial] | 2m17s
: OCP-31276:sgao:Windows_Containers:[sig-windows] Windows_Containers Configure CNI and internal networking check | 2m43s
: OCP-33783:sgao:Windows_Containers:[sig-windows] Windows_Containers Enable must gather on Windows node [Slow][Disruptive] [Serial] | 6m51s
: OCP-37472:sgao:Windows_Containers:[sig-windows] Windows_Containers Idempotent check of service running in Windows node [Slow][Disruptive] [Serial] | 2m18s
: OCP-60814:rrasouli:Windows_Containers:[sig-windows] Windows_Containers Check containerd version is properly reported | 6s
: OCP-70922:weinliu:Windows_Containers:[sig-windows] Windows_Containers Monitor CPU, Memory, and Filesystem graphs for Windows Pods managed by wmco | 4s
: OCP-32615:sgao:Windows_Containers:[sig-windows] Windows_Containers Generate userData secret [Serial] | 1m20s
: OCP-42204:rrasouli:Windows_Containers:[sig-windows] Windows_Containers Create Windows pod with a Projected Volume | 6m55s
: OCP-28632:sgao:Windows_Containers:[sig-windows] Windows_Containers Windows and Linux east west network during a long time | 6s
: OCP-25593:sgao:Windows_Containers:[sig-windows] Windows_Containers Prevent scheduling non Windows workloads on Windows nodes

**Comment by weinliu** [comment]
@rrasouli could you help take a look? Thanks

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -6027,4 +6058,4 @@ user_secrets_target_clusters:
 - build11
 - hosted-mgmt
 - vsphere02
-vault_dptp_prefix: dptp
+vault_dptp_prefix: dptp
```
Comment (rrasouli) [inline core-services/ci-secret-bootstrap/_config.yaml:6061]: Add new line here

**Comment by rrasouli** [comment]
The job configuration files should not be modified directly as they are auto-generated. Instead:

1. Make your intended changes in the ci-operator configuration files
2. Run `make jobs` to regenerate the job configurations
3. Commit the regenerated files

This ensures consistency and follows the project's automation practices. If you're trying to remove the debug-winc-vsphere-ipi test, please make that change in the source configuration files instead.ih0iEW1M

File: core-services/ci-secret-bootstrap/_config.yaml (hunk 1 above)
Comment (weinliu) [reply core-services/ci-secret-bootstrap/_config.yaml]: updated

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 2]
```diff
@@ -6028,3 +6059,4 @@ user_secrets_target_clusters:
 - hosted-mgmt
 - vsphere02
 vault_dptp_prefix: dptp
+
```
Comment (rrasouli) [inline core-services/ci-secret-bootstrap/_config.yaml:6062]: Remove this line

Comment (weinliu) [reply core-services/ci-secret-bootstrap/_config.yaml]: Removed now, I'm confulsed, the checking asked for a line earlier...

Comment (weinliu) [reply core-services/ci-secret-bootstrap/_config.yaml]: @rrasouli It looks good now

**Comment by weinliu** [comment]
/uncc @kuiwang02 
/uncc @hongkailiu 
@liangxia could you help take a look?

**Comment by weinliu** [comment]
@jcpowermac @jhou1 @Prucek could you help take a look? Thanks!

**Comment by smg247** [comment]
@weinliu this is causing the ci-secret-bootstrap job to fail:
```
"config.245.\"pull-secret\": couldn't get email field 'email' from item dptp/quay.io/multi-arch: item at path \"kv/dptp/quay.io/multi-arch\" has no key \"email\"
```
I am going to remove the email field from the config.

File: ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ ref:
     - name: WINDOWS_NODE_REPLICAS
       default: "1"
     - name: WINDOWS_OS_ID
-      default: "windows-golden-images/windows-server-2022-template-qe"
+      default: "windows-server-2022-template-qe-20241104"
```
Comment (rrasouli) [inline ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml:18]: I was thinking this image not under folder can get pruned

Comment (weinliu) [reply ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml:18]: I'll fix this. It's actually under windows-golden-images/

**Pull request merged**